hcert_spec.md

```diff
@@ -86,48 +89,57 @@ The Claim Key 1 is used to identify this claim.
 
 The Expiration Time (**exp**) claim SHALL hold a timestamp in the NumericDate format (as specified in [RFC 8392](https://tools.ietf.org/html/rfc8392) section 2) indicating for how long this particular signature over the Payload SHALL be considered valid, after which a Verifier MUST reject the Payload as expired. The purpose of the expiry parameter is to force a limit of the validity period of the EHC. The Claim Key 4 is used to identify this claim.
 
+The Expiration Time MUST not exceed the validity time in the DSC.  The verifier SHOULD check this.
 
 #### Issued At
 
-The Issued At (**ia**t) claim SHALL hold a timestamp in the NumericDate format (as specified in [RFC 8392](https://tools.ietf.org/html/rfc8392) section 2) indicating the time when the EHC was created. Verifiers MAY apply policies with the purpose of restricting the validity of the EHC based on the time of issue. The Claim Key 6 is used to identify this claim.
+The Issued At (**ia**t) claim SHALL hold a timestamp in the NumericDate format (as specified in [RFC 8392](https://tools.ietf.org/html/rfc8392) section 2) indicating the time when the EHC was created. 
+
+The Issued At  MUST not predate the validity time in the DSC.  The verifier MAY check this.
```

Comment:
```suggestion
The Issued At MUST not predate the validity time in the DSC.  The verifier MAY check this.
```

Same comment as above, but the risks of negative consequences are likely smaller. But I am reluctant to adding validation rules which contradicts the basic principle of the trust list. However, a verifier MAY or even SHOULD check that ```iat``` is not in the future?